src/components/pageTemplates/NodeBasicPage.tsx

```diff
@@ -37,6 +37,9 @@ function NodeBasicPage({ node, sidebar, ...props }: NodeBasicPageProps): JSX.Ele
         </div>
         <div className="columns">
           <div className="lower-content-region col col-12">
```

Comment:
The previously added col class could be removed here because it's just adding extra margin that is not needed with col-12.

Reply:
col-12 is needed to give 100% layout for the wrapper. Will break without.